html_css/floats.md

```diff
@@ -0,0 +1,39 @@
+### Introduction
+While floats are currently used predominantly for wrapping text around images it's worthwhile to understand what they were previously used for, and why we should steer away from using them for other purposes.
+
+### Learning Outcomes
+* You'll learn how to properly use floats. 
+* You'll learn why to avoid using floats the way they were used in the past, and their history.
```

Comment:
* Learn how floats work and how to use them
* Understand how floats were used in the past, and why to avoid these methods in the present